pbst.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2017 CN-Consult GmbH
+ * @author Jens Stahl <jens.stahl@cn-consult.eu>
+ *
+ * License: Please check the LICENSE file for more information.
+ */
+
+require_once ("vendor/autoload.php");
+foreach (glob(getcwd()."/commands/*command.php") as $filename)
+{
+    require_once ($filename);
+}
+require_once ('profitbricksapi/profitbricksapi.php');
+require_once ('profitbricksapi/datacenter.php');
+require_once ('profitbricksapi/virtualmachine.php');
+require_once ('profitbricksapi/virtualdisk.php');
+require_once ('profitbricksapi/snapshot.php');
+
+use Symfony\Component\Console\Application;
+
+$application = new Application();
+$application->add(new \PBST\Command\SnapshotListCommand());
```

Comment:
There exists a minor naming convention problem with the used namespaces and the directory structure:
\PBST\Command\SnapshotDeleteCommand() should be located in an additional subfolder PBST
like it is organized with the contained classes in the other subdirectory **ProfitBricksApi**
For example: `class ProfitBricksApi in \ProfitBricksApi\ProfitBricksApi`